kibana/Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV AOP_KIBANA_PLUGIN_VER=4.5.1 \
     CONFIG_PATH=/etc/kibana \
     NODE_ENV=production \
     KIBANA_VER=4.6.4 \
-    NODE_BIN=/usr/bin/node \
+    NODE_BIN=/opt/rh/rh-nodejs6/root/usr/bin/node \
```

Comment:
You are correct it is still pulling in the nodejs specified in the spec file.  We need to changes that as well.  I just need to figure out how to do that.  I tested the build of the image by creating a scratch build through OSBS, basically what is described in the 'building images' section.  This produced the image found here [2].

[1] https://mojo.redhat.com/docs/DOC-1137707
[2] https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=13768194